tests/components/workday/test_binary_sensor.py

```diff
@@ -12,6 +12,12 @@
 
 FUNCTION_PATH = "homeassistant.components.workday.binary_sensor.get_date"
 
+DEFAULT_CONFIG = {
+    "platform": "workday",
+    "add_holidays": [],
+    "remove_holidays": [],
+}
+
```

Comment:
`add_holidays` and `remove_holidays` are set to `ensure_list` in schema validation